fuse/data/ops/op_base.py

```diff
@@ -68,8 +89,28 @@ def reverse(self, sample_dict: NDict, key_to_reverse: str, key_to_follow: str, o
         :param key_to_follow: run the reverse according to the operation applied on this value
         :return: modified sample_dict
         """
-        raise NotImplemented
+        raise sample_dict
 
     
+def op_call(op: OpBase, sample_dict: NDict, op_id: str, **kwargs):
+    try:
+        if isinstance(op, OpReverseableBase):
+            return op(sample_dict, op_id=op_id, **kwargs)
+        else: # OpBase but note reversable
```

Comment:
should we explicitly check for OpBase in the other option, for the case that someone mistakenly provided something different? 
(and raise exception if it's neither?)

Reply:
I want to keep the option to use callables  instead